docs/en/05_Contributing/02_Request_for_comment.md

```diff
@@ -0,0 +1,46 @@
+# Request for comment (RFC)
+
+## Why RFCs?
+This is part of the SilverStripe core decision-making process and addresses the gap between the idea phase and the pull request submission (and merge).
+
+The rationale behind this process is to:
+ * Improve visibility on how the decisions made, producing a more open community mood
+ * Clarity on what the proposal is, its rationale and impact
+ * Reduce unnecessary work when coding large features is done without community and core-committers buy-in
+ * Improved likelihood of an optimal solution being merged - the pressure is much higher to approve the change if the coding has already been done (it’s much easier to make adjustments in the design phase)
+
+The important thing to understand about the RFCs is that these are NOT a way to request features. Rather, they are a way to bring clarity so people understand what change is being proposed.
+
+## When to write an RFC?
+
+We intend RFCs to be the primary mechanisms for proposing major new features, for collecting community input on an issue, and for documenting the design decisions that have gone into SilverStripe. The RFC author is responsible for building consensus within the community and documenting dissenting opinions.
+
+Before writing the actual summary RFC document, the idea should have already had a wide range of discussion in various community communication channels. Once discussions reach a point where you think most of the difficulties have been worked through then create the RFC using the template provided.
+
+The benefits of writing an RFC for non-trivial feature proposals are:
+ * Obtaining a preliminary approval from core-committers on an architecture before code is completed, to mitigate the risk of a non-merge after a PR is submitted
+ * Community becomes aware of incoming changes prior to the implementation
+ * RFC can be used as a basis for documentation of the feature
+	
+## How to write an RFC?
+### Template
+The following heading can act as a template to starting your RFC.
+ * **Introduction** - include a reference #, title, author
+ * **Motivation** - why this is a good idea
```

Comment:
I'd say the first two items should be standardised metadata, and the second item before the motivation should be a "Purpose and outcome". Otherwise the document lacks the overview statement. I've added such a section to [RFC-1](https://github.com/silverstripe/silverstripe-framework/issues/3792) now.

Maybe something like this:

* Metadata - standardised header containing at least the Author(s), Status and Version fields.
* Purpose and outcome - the purpose of this document, and the expected outcome.
* ...and then continued starting from Motivation